src/ccalls.jl

```diff
@@ -136,17 +136,17 @@ whereas `x` in the latter example lives in the Julia session.
 """
 function evalstr(cmd::String)
     res = evalstr_ex(cmd * ";")
-    if any(x->x[1] == false, res)
+    if any(x::GapObj->x[1] === false, res)
```

Comment:
I read this as the recommendation to use always `===` or `!==` when comparing something with `true`, `false`, `GAP.Globals.fail`.

Reply:
Yeah. It at least ensures that the compiler can always turn this into the most optimal code; here, it can't predict the type of `x[1]` so it should check at runtime whether there is a `==` method for whatever `x[1]` is plus a `Bool`... But we have extra information about the intent of the code, so that `===` is what we want, and can totally away runtime dispatch here.